### PR TITLE
iOS source map support

### PR DESCRIPTION
**Pull request by konraddysput**
# Why

This diff simplifies Backtrace source map support for iOS in react-native.

ref: BT-761

### CODE REVIEW

**Review by rick-bt — APPROVED** [review]

File: examples/sdk/reactNative/README.md [hunk 1]
```diff
@@ -19,3 +24,38 @@ android-sourcemap.sh script.
...
+# if you dont use hermes support, please skip this step.
+export HERMES_CLI_PATH="$(pwd)/../ios-hermesc.sh"
+
+/bin/sh -c "$WITH_ENVIRONMENT $REACT_NATIVE_XCODE"
+
+# copy javascript build output to the build directory
+cp "$CONFIGURATION_BUILD_DIR/main.jsbundle" $SOURCE_MAP_DIR
+
+# process source map with javascript code
+backtrace-js run --config "$(pwd)/../.backtracejsrc" --path "$SOURCE_MAP_DIR/main.jsbundle"
```
Comment (perf2711) [inline examples/sdk/reactNative/README.md:56]: maybe we should use `npx` here? 

```
npx --yes backtrace-js run --config "$(pwd)/../.backtracejsrc" --path "$SOURCE_MAP_DIR/main.jsbundle"
```

(the `--yes` switch will install `backtrace-js` if it is not available).

File: examples/sdk/reactNative/README.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ This example app shows features available in the @backtrace/react-native package
 
 #### Source maps
 
+Before executing any step:
+
+-   Please update .backtracejsrc file with your symbols submission URL and your sourcemap settings.
+-   please install @backtrace/javascript-cli library.
```
Comment (perf2711) [inline examples/sdk/reactNative/README.md:16]: AFAIK the user would have to install this package globally for line 56 to work. Could we use `npx` instead?

Comment (konraddysput) [reply examples/sdk/reactNative/README.md:16]: Good idea!

File: examples/sdk/reactNative/README.md (hunk 1 above)
Comment (konraddysput) [reply examples/sdk/reactNative/README.md:56]: Good idea!

**Review by perf2711 — APPROVED** [review]

**Pull request merged**